### DISCUSSION TITLE
Ability for nature to create its own vegetation again.

### DISCUSSION THREAD

**Discussion by JustLearninglol** [niels747/2D-Weather-Sandbox#30] [Ideas]
So currently, we have to manually put vegetation there and while thats not bad, if my suggestion for lightning fires is put into consideration, the rain from storms should be able to create vegetation when it falls so nature balances it out. It should also dry out but not dissapear, like it dies and it can be fuel for fires. Fires. :)

**Comment by niels747** [comment]
Okay, I will add it as an option. Either dynamic or static vegetation.
